persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/CancerTypeMapper.java

```diff
@@ -0,0 +1,4 @@
+package org.cbioportal.persistence.mybatis;
+
+public interface CancerTypeMapper {
```

Comment:
In the `DETAILED` projection of Clinical Data, we return patient,study and cancer type information too, and instead of writing select statement for those tables in every MyBatis script, I reuse them from their corresponding script using the `include` tag. But MyBatis doesn't see those scripts unless I create an interface for them. So they are empty for now but they are required to use sql fragments from other scripts.